src/middleware/gateway.service.ts

```diff
@@ -31,6 +32,22 @@ export class GatewayService {
       return response.data;
     } catch (error) {
       if (error.response) {
+        if (changeResponse) {
+          if (
+            error.response.data.params.err ===
+              'ERR_YOUTUBE_LICENSE_VALIDATION' &&
+            url ===
+              'https://content-dev.prathamdigital.org/asset/v4/validate?field=license'
```

Comment:
@arati-tekdi can you get this URL from env ?

Reply:
no. this is handling for one API. In future we will make that as generic. For now it is only for particular error, therefor i have added like this.